packages/babel-polyfill/package.json

```diff
@@ -8,7 +8,7 @@
   "repository": "https://github.com/babel/babel/tree/master/packages/babel-polyfill",
   "main": "lib/index.js",
   "dependencies": {
-    "core-js": "^1.0.1",
+    "core-js": "^2.0.0",
```

Comment:
If this was deemed a breaking change for Core-js, would that make it a breaking change for Babel too, or are the breaking changes in parts that aren't on by default in Babel?